pkg/action/executor/rescheduler.go

```diff
@@ -85,45 +85,79 @@ func (r *ReScheduler) checkActionItem(action *proto.ActionItemDTO) error {
 	return nil
 }
 
-// get k8s.nodeName of the node
-// TODO: get node info via EntityProperty
+// get k8s.node of the new hosting node
 func (r *ReScheduler) getNode(action *proto.ActionItemDTO) (*api.Node, error) {
 	hostSE := action.GetNewSE()
 
 	var err error = nil
 	var node *api.Node = nil
 
-	if r.stitchType == stitching.UUID {
-		node, err = util.GetNodebyUUID(r.kubeClient, hostSE.GetId())
-	} else if r.stitchType == stitching.IP {
-		var machineIPs []string
-		if hostSE.GetEntityType() == proto.EntityDTO_VIRTUAL_MACHINE {
-			vmData := hostSE.GetVirtualMachineData()
-			if vmData == nil {
-				err := fmt.Errorf("Missing virtualMachineData[%v] in targetSE.", hostSE.GetDisplayName())
-				glog.Error(err.Error())
-				return nil, err
-			}
-			machineIPs = vmData.GetIpAddress()
+	//0. check entity type
+	etype := hostSE.GetEntityType()
```

Comment:
In some case, hostSe can be nil. Can it happen here?

Reply:
Yea, possible, if mediation server has bug. I will check this.